modules/ietf-udp-client.yang

```diff
@@ -0,0 +1,104 @@
+module ietf-udp-client {
+  yang-version 1.1;
+  namespace
+    "urn:ietf:params:xml:ns:yang:ietf-udp-client";
+  prefix udpc;
+  import ietf-inet-types {
+    prefix inet;
+    reference
+      "RFC 6991: Common YANG Data Types";
+  }
+  /*import ietf-tls-client {
+    prefix tlsc;
+    reference
+      "RFC TTTT: YANG Groupings for TLS Clients and TLS Servers";
+  }*/
```

Comment:
Please put this into a separate commit with a commit message that explains why it needs to be done. Also, please add these YANG models in an extra commit which clearly states where they were taken from, which RFC/draft it is.